Doc/library/sched.rst

```diff
@@ -69,7 +69,7 @@ Scheduler Objects
    Schedule a new event. The *time* argument should be a numeric type compatible
    with the return value of the *timefunc* function passed  to the constructor.
    Events scheduled for the same *time* will be executed in the order of their
-   *priority*.
+   *priority*. A lower number represents a higher priority.
```

Comment:
If the former wording was unclear, wouldn't be better to add just a word "increasing": `executed in the order of increasing their *priority*`. "Lower is higher" looks slightly orwellish to me.

Reply:
@serhiy-storchaka So we should invert the meaning?